src/PharoLauncher-CLI-Tests/PhLImageCopyCliCommandTest.class.st

```diff
@@ -0,0 +1,53 @@
+Class {
+	#name : #PhLImageCopyCliCommandTest,
+	#superclass : #PhLImageCliCommandTest,
+	#instVars : [
+		'process',
+		'imageDir'
+	],
+	#category : #'PharoLauncher-CLI-Tests'
+}
+
+{ #category : #tests }
+PhLImageCopyCliCommandTest >> testCopyImageNameAlreadyUsed [
+	self addImageNamed: 'myImage1'.
+	self addImageNamed: 'myImage2'.
+
+	self runCommand: #('launcher' 'image' 'copy' 'myImage1' 'myImage2' ).
+	 
+	self assert: self errorString equals:''.  
```

Comment:
In a real terminal with the real command  I just added an error message when the name is already used, but since the mock defines a new copy (assuming the existing copy has already been tested) there are no checks if the name is available.